app/components/cards/PostSummary.jsx

```diff
@@ -149,7 +149,7 @@ class PostSummary extends React.Component {
         let thumb = null;
         if(pictures && p.image_link) {
           const prox = $STM_Config.img_proxy_prefix
-          const size = (thumbSize == 'mobile') ? '640x480' : '128x256'
+          const size = (thumbSize == 'mobile') ? '640x480' : '256x384';
```

Comment:
Here's a few examples as well(you can load them side by side)

https://img1.steemit.com/256x584/http://www.impawards.com/2016/posters/moana_ver5.jpg
https://img1.steemit.com/256x384/http://www.impawards.com/2016/posters/moana_ver5.jpg

https://img1.steemit.com/256x584/http://i.imgsafe.org/501676a6b1.jpg
https://img1.steemit.com/256x384/http://i.imgsafe.org/501676a6b1.jpg